server/sonar-server/src/test/java/org/sonar/server/permission/ws/RemoveGroupActionTest.java

```diff
@@ -316,7 +317,11 @@ public void removing_project_permission_is_allowed_to_project_administrators() t
   }
 
   private void loginAsAdmin() {
-    userSession.login().setGlobalPermissions(SYSTEM_ADMIN);
+    loginAsOrganizationAdmin(db.getDefaultOrganization());
+  }
+
+  private void loginAsOrganizationAdmin(OrganizationDto org) {
```

Comment:
I think a unit test is missing using this method with an organization which is not the default one